src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -255,12 +255,21 @@ internal void WorkerInitResponse(GrpcEvent initEvent)
             _workerChannelLogger.LogDebug("Received WorkerInitResponse. Worker process initialized");
             _initMessage = initEvent.Message.WorkerInitResponse;
             _workerChannelLogger.LogDebug($"Worker capabilities: {_initMessage.Capabilities}");
+
+            if (_initMessage.WorkerMetadata != null)
+            {
+                _initMessage.WorkerMetadata = ParseWorkerMetadata(_initMessage.WorkerMetadata);
+                _metricsLogger.LogEvent(MetricEventNames.WorkerMetadata, functionName: null, _initMessage.WorkerMetadata.ToString());
```

Comment:
Do you have an example of what the serialized payload will look like with custom properties? It would be good to understand whether the way we'll be persisting this will make it easy/efficient for querying.

Reply:
@fabiocav yup, there's a screenshot in the PR description - it's a json object so should be easy to query